packages/tsconfig/docs/examples.md

```diff
@@ -142,70 +133,12 @@ And then in your build steps:
 1. Output JS:
    `tsc`
 1. Output declaration files:
-   `tsc -d --emitDeclarationOnly --allowJs false --declarationDir dist`
+   `tsc -d --emitDeclarationOnly --allowJs false --declarationDir dist --declarationMap`
```

Comment:
This two-step process is technically not required with the newest version of TS, which can output declaration files while `--allowJs true`!

I wonder if we need these two sections at all. Perhaps the sections should be:

- web-scripts build
- manually configuring tsconfig extends to output a "reasonable default" of es6 + polyfills + allowing JS.

Reply:
Agreed, simplifies things a lot and avoids outdated configs.